e_os.h

```diff
@@ -297,7 +297,7 @@ struct servent *getservbyname(const char *name, const char *proto);
 
 /* ----------------------------- HP NonStop -------------------------------- */
 /* Required to support platform variant without getpid() and pid_t. */
-# ifdef __TANDEM
+# ifdef _GUARDIAN_TARGET
```

Comment:
Is it safe to drop `__TANDEM`?

Reply:
So you're saying that you know for a fact that those macros will never appear on another platform?

Reply:
Well, there is that. Ok, putting it back.

Reply:
20+ years of dealing with mixtures of diverse unices, Windows and VMS brings on a bit of ...  let's call it slight paranoia :wink: